nautobot/core/settings.py

```diff
@@ -569,6 +571,11 @@
 }
 
 CONSTANCE_CONFIG = {
+    "ALLOW_REQUEST_PROFILING": ConstanceConfigItem(
+        default=True,
```

Comment:
Looking at this from an administrator's perspective, I'd probably want this to be disabled by default and only enable it when I want a user to generate data for me. Otherwise users may turn this on accidentally or maliciously and put extra load on the system.

Reply:
Yeah I'm fine that, given it is a constance setting.